docs/evaluate/use-cases-design-patterns.mdx

```diff
@@ -0,0 +1,111 @@
+---
+id: use-cases-design-patterns
+title: Temporal Use Cases and Design Patterns 
+description: Discover various use cases and architectural design patterns for implementing Workflows with Temporal.
+sidebar_label: Use Cases
+tags:
+  - temporal-use-cases
+  - temporal-design-patterns
+keywords:
+  - temporal use cases
+  - temporal design patterns
+  - temporal workflows
+---
+
+This page provides an overview of how leading organizations leverage Temporal to solve real-world problems, general use cases, and architectural design patterns. 
+
+## Use Cases of Temporal in Production
+
+Here are some examples where Temporal is most impactful and running in production at large organizations today.
```

Comment:
```suggestion
Here are some examples where Temporal is most impactful and running in production at large organizations today.  For more examples, see our [Temporal Use Cases](https://temporal.io/in-use) page.

If you're interested in code to help get you started, check our [Temporal Example Applications](https://learn.temporal.io/examples/).
```

Reply:
These were my two questions I had when reading this:

1. Where do I find more of these?
2. How do I get started if I have one of these use cases?